apps/trading/client-pages/market/market-header-stats.tsx

```diff
@@ -53,7 +53,7 @@ export const MarketHeaderStats = ({ market }: MarketHeaderStatsProps) => {
 
   return (
     <>
-      <HeaderStat heading={t('Mark Price')} testId="market-price">
+      <HeaderStat heading={t('Price')} testId="market-price">
```

Comment:
I don't think we should change this one as my incoming Pr for handling spot markets in the header adds a new header stat for last price and it is label as such. So for futures and perps I think its preferable to have the label more accurately say 'Mark price'